.github/workflows/self-service-labeler.yml

```
@@ -0,0 +1,40 @@
+name: Issue Opened Self-Service Labeler
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  add-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - name: Checkout magic modules
+      uses: actions/checkout@v2
+      with:
+        repository: GoogleCloudPlatform/magic-modules
+        ref: main
+        path: magic-modules
+    - name: Setup go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.19.0'
+    - name: Pick labels
+      id: pick_labels
+      env:
+        ISSUE_BODY: ${{ github.event.issue.body }}
+      run: |
+        cd magic-modules/tools/issue-labeler
+        echo "labels=$(go run .)" >> $GITHUB_OUTPUT
+    - name: Apply labels
+      uses: actions/github-script@v6
```

```suggestion
      uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6.4.1
```

This is a different version (6.0.0 => 6.4.1). At HashiCorp the 'approved' versions of this GHA are currently v6.4.0 and v6.4.1. Either I can go through the process of getting v6.0.0 approved or we can upgrade the version we use here to match one that's already approved.

I've just made the same change for a different GHA in this repo, here: https://github.com/hashicorp/terraform-provider-google/pull/15194